src/main/java/app/coronawarn/verification/model/Tan.java

```diff
@@ -39,14 +40,15 @@
 @Data
 @AllArgsConstructor
 @RequiredArgsConstructor
+@NoArgsConstructor
 public class Tan {
 
   @NotNull
+  @NonNull
```

Comment:
Is Lombocks @NonNull neccessary?

Reply:
Yes ist is neccessary for lombock requiredargs